flutter/lib/src/default_integrations.dart

```diff
@@ -367,8 +367,15 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
       if (!kIsWeb) {
         if (options.release == null || options.dist == null) {
           final packageInfo = await _packageLoader();
+          var name = packageInfo.packageName;
+          if (name.isEmpty) {
+            // Not all platforms have a packageName
+            // https://github.com/getsentry/sentry-dart/issues/410
```

Comment:
instead of linking a GH issue, I'd rather describe the problem/solution as a comment, eg:
// not all platforms have a `packageName`, if not, take `appName` instead